### ISSUE TITLE
TypeError: `client` must be a discord.js Client instance

### ISSUE THREAD

**Issue by Dqrshan** [Dqrshan/discord-debug#1]
```ts
throw new TypeError('`client` must be a discord.js Client instance');
            ^

TypeError: `client` must be a discord.js Client instance
```

This error only occurs in >1.0.8 versions. There is no other error being produced, and is hard to locate what exactly is producing this error. 

Contributions & fixes are greatly appreciated.

**Comment by Dqrshan** [comment]
**Update**:
- Using the module after build in the root directory seems to produce no errors and works as expected.
- Installing the module via `npm` or external folder (outside root dir) produces the error.

**Comment by Dqrshan** [comment]
**Update**:

This issue is asked as a question in the link below: 
- https://stackoverflow.com/questions/76551320/instanceof-returns-false-on-a-true-condition

**Comment by Dqrshan** [comment]
**Update**:

I will be rewriting the complete package from scratch. 
- Rewrite branch: [dev](https://github.com/Dqrshan/discord-debug/tree/dev)

**Comment by Dqrshan** [comment]
**Fixed!**

This error is now fixed in **v2.0.5**
```bash
npm i discord-debug@latest
```
